portal/server/src/config.js

```diff
@@ -1,7 +1,7 @@
 export const {
     PORT = 5001,
     NODE_ENV = 'development',
-    MONGO_URI = 'mongodb://mongodb:27017/qStudy_db',
+    MONGO_URI = 'mongodb://qStudy_root:bigcamp93@mongodb:27017/qStudy_db?authSource=admin',
```

Comment:
dude don't do this. Instead keep it in environment variables and we'll not commit that file